README.md

```diff
@@ -1 +1,57 @@
-# To_Do_List
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# To_Do_List
+
+
+> "To-do list" is a tool created to help you orginize your daily activities by keeping them listed that you can consult whenever you want, with a feature of marking them as complete. I'm building this To-do list with Javascript (ES6) and Webpack.
+
+## Built With
+
+- Javascript
+- HTML and CSS
+- webpack
+
+
+## Getting Started
+
+You can have a copy of this repo run on your local machine by just directly cloning it.
```

Comment:
### OPTIONAL
- [ ] Kindly add directions and commands on how to clone this repository and get this project running locally 👍🏿